app/Http/Controllers/Forum/TopicsController.php

```diff
@@ -157,17 +168,23 @@ public function show($id) {
 			abort($skipLayout ? 204 : 404);
 		}
 
+		$isDoublePost = $this->isDoublePost($lastPost = $posts[count($posts) - 1]);
 		$postsPosition = $topic->postsPosition($posts);
 
 		Event::fire(new TopicWasViewed($topic, $posts->last(), Auth::user()));
 
 		$template = $skipLayout ? "_posts" : "show";
 
-		return view("forum.topics.{$template}", compact("topic", "posts", "postsPosition", "jumpTo"));
+		return view("forum.topics.{$template}", compact("topic", "posts", "postsPosition", "jumpTo", 'isDoublePost'));
```

Comment:
Simple strings are replaced with single quotes by StyleCI. Click details and apply the patch that StyleCI provides (below) after rebasing against master to automatically fix your files.